orientdb/src/main/java/site/ycsb/db/OrientDBClient.java

```diff
@@ -276,48 +368,59 @@ public Status update(String table, String key, Map<String, ByteIterator> values)
     }
   }
 
-  @Override
-  public Status scan(String table, String startkey, int recordcount, Set<String> fields,
-      Vector<HashMap<String, ByteIterator>> result) {
+  private String preparedUpdateSql(final String table, final Map<String, ByteIterator> values) {
+    return "UPDATE "
+        + table
+        + " SET "
+        + values.entrySet().stream()
+            .map(e -> " " + escapeQuotes(e.getKey()) + "= '" + escapeQuotes(e.getValue().toString()) + "'")
+            .collect(Collectors.joining(", "))
+        + " WHERE key = :key";
+  }
 
-    if (isRemote) {
-      // Iterator methods needed for scanning are Unsupported for remote database connections.
-      LOG.warn("OrientDB scan operation is not implemented for remote database connections.");
-      return Status.NOT_IMPLEMENTED;
-    }
+  private String escapeQuotes(final String key) {
+    // Brutal
+    return key.replace("\"", "").replace("'", "").replace("\\", "");
+  }
```

Comment:
add a javadoc to this method on what it's doing?

Reply:
done - now called `escapeUnsupportedChars`